README.md

```diff
@@ -39,18 +39,19 @@ Funiq can read from a file or have its input piped from stdin.
 
 	USAGE: 
 
-     funiq  [-c] [-t] [-a] [-i] [-d <integer>] [--] [--version] [-h]
+     funiq  [-c] [-a] [-i] [-I] [-d <integer>] [--] [--version] [-h]
```

Comment:
Hey - would you be able to update the examples above this in the README to use the new `-I` flag instead of `-c`?

Reply:
Will do, I'll push changes after work